src/components/notebook/NotebookCard.tsx

```diff
@@ -54,13 +54,13 @@ const InnerCard = memo(
     const [answerToggled, setAnswerToggled] = useState(false);
 
     return (
-      <Paper
-        p="md"
-        className={classes.card}
-        onClick={() => setAnswerToggled(!answerToggled)}
-      >
+      <Paper p="md" className={classes.card}>
         <Group align="top" justify="space-between" wrap="nowrap">
-          <Group align="center" w="100%">
+          <Group
+            align="center"
+            w="100%"
+            onClick={() => setAnswerToggled(!answerToggled)}
```

Comment:
Nitpick: I know it's a bit less bare react, but mantine's useDisclosure is nice for those case

you could then (skipping the function re-creation at every render):
```suggestion
            onClick={handlers.toggle}
```